src/main/kotlin/com/github/evgeniikuznetsov/drill4jplugin/actions/FileRetrieveAction.kt

```diff
@@ -1,27 +1,56 @@
 package com.github.evgeniikuznetsov.drill4jplugin.actions
 
-import com.github.evgeniikuznetsov.drill4jplugin.config.SettingsState
-import com.github.evgeniikuznetsov.drill4jplugin.tools.UrlFileRetriever
-import com.intellij.codeInspection.ex.GlobalInspectionContextImpl
+import com.github.evgeniikuznetsov.drill4jplugin.config.*
+import com.github.evgeniikuznetsov.drill4jplugin.tools.*
+import com.intellij.codeInspection.ex.*
 import com.intellij.coverage.*
-import com.intellij.openapi.actionSystem.AnAction
-import com.intellij.openapi.actionSystem.AnActionEvent
-import com.intellij.openapi.actionSystem.CommonDataKeys
-import com.intellij.openapi.progress.ProgressIndicator
-import com.intellij.openapi.progress.ProgressManager
-import com.intellij.openapi.progress.Task
-import org.apache.commons.lang3.StringUtils
-import java.io.File
+import com.intellij.openapi.actionSystem.*
+import com.intellij.openapi.progress.*
+import com.intellij.openapi.project.*
+import java.io.*
+import java.nio.file.*
 
 class FileRetrieveAction : AnAction() {
 
-    override fun actionPerformed(e: AnActionEvent) {
+    override fun actionPerformed(event: AnActionEvent) {
+        if (!SettingsState.settings.fromLocalFile) {
+            getCoverageFromLocalFile(event)
```

Comment:
if SettingsState.settings.fromLocalFile then getCoverageFromLocalFile(event) ?
```suggestion
        if (SettingsState.settings.fromLocalFile) {
            getCoverageFromLocalFile(event)
```